src/containers/chat/list-of-users-with-search/ListOfUsersWithSearch.constants.ts

```diff
@@ -1,13 +1,23 @@
 import { ChatResponse } from '~/types'
 
+const sortChatsByDate = (listOfChats: ChatResponse[]): ChatResponse[] => {
+  return listOfChats.sort((a, b) => {
+    const dateA = new Date(a.latestMessage.createdAt).getTime()
```

Comment:
Not latestMessage as it was in you previous PR
Because we can save chats without any messages (new chat)
Use, 'updatedAt' field for sort purpose, please
![image](https://github.com/user-attachments/assets/d7c7fb46-0a8a-4651-8500-18710bc0483a)